appveyor.yml

```diff
@@ -13,112 +13,116 @@ artifacts:
 
 build_script:
 
-- cmd: >-
-    rem Set up DT_* environment variables
+- ps: >-
 
-    rem =======================================================================
+    # Set up required variables
 
-    SET DT_HARNESS = AppVeyor
+    # =======================================================================
 
+    $env:DT_HARNESS = "AppVeyor"
 
-    FOR /f %%i in ('git rev-list --count main') DO SET MAIN_BUILD_ID=%%i
+    $DEFAULT_PATH = $env:PATH
 
-    rem AppVeyor does an additional merge commit, that we don't need to take into account
+    $MAIN_BUILD_ID = git rev-list --count main
 
-    FOR /f %%i in ('git rev-list --count main..') DO SET /a "BRANCH_BUILD_ID=%%i-1"
 
+    # Do not take into account the AppVeyor's additional merge commit
 
-    echo APPVEYOR_REPO_BRANCH = %APPVEYOR_REPO_BRANCH%
-
-    echo APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH = %APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH%
-
-    echo APPVEYOR_PULL_REQUEST_NUMBER = %APPVEYOR_PULL_REQUEST_NUMBER%
-
-    echo MAIN_BUILD_ID = %MAIN_BUILD_ID%
-
-    echo BRANCH_BUILD_ID = %BRANCH_BUILD_ID%
-
-- ps:
+    $BRANCH_BUILD_ID = (git rev-list --count main..) - 1
 
     if ($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH -match "^rel-") {
-      $env:DT_RELEASE="True"
+      $env:DT_RELEASE = "True"
     } elseif ($env:APPVEYOR_PULL_REQUEST_NUMBER -ne $null) {
-      $env:DT_BUILD_SUFFIX="PR$env:APPVEYOR_PULL_REQUEST_NUMBER.$env:BRANCH_BUILD_ID"
+      $env:DT_BUILD_SUFFIX = "PR$env:APPVEYOR_PULL_REQUEST_NUMBER.$BRANCH_BUILD_ID"
     } elseif ($env:APPVEYOR_REPO_BRANCH -eq "main") {
-      $env:DT_BUILD_NUMBER=$env:MAIN_BUILD_ID
+      $env:DT_BUILD_NUMBER = $MAIN_BUILD_ID
     } else {
-      $env:DT_BUILD_SUFFIX="$env:APPVEYOR_REPO_BRANCH$env:BRANCH_BUILD_ID"
+      $env:DT_BUILD_SUFFIX = "$env:APPVEYOR_REPO_BRANCH.$BRANCH_BUILD_ID"
     }
 
-- cmd: >-
 
-    echo DT_RELEASE      = %DT_RELEASE%
+    # Print out variable's content
 
-    echo DT_BUILD_NUMBER = %DT_BUILD_NUMBER%
+    # =======================================================================
 
-    echo DT_BUILD_SUFFIX = %DT_BUILD_SUFFIX%
+    echo "DEFAULT_PATH = $DEFAULT_PATH"
 
+    echo "APPVEYOR_REPO_BRANCH = $env:APPVEYOR_REPO_BRANCH"
 
-    rem Python 3.5
+    echo "APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH = $env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH"
 
-    rem =======================================================================
+    echo "APPVEYOR_PULL_REQUEST_NUMBER = $env:APPVEYOR_PULL_REQUEST_NUMBER"
 
-    SET PATH=C:\Python35-x64;C:\Python35-x64\Scripts;%PATH%
+    echo "MAIN_BUILD_ID = $MAIN_BUILD_ID"
 
-    python ci\ext.py wheel
+    echo "BRANCH_BUILD_ID = $BRANCH_BUILD_ID"
 
-    FOR %%i in (dist\*-cp35-*.whl) DO pip install %%i
+    echo "DT_RELEASE = $env:DT_RELEASE"
 
-    pip install pytest docutils pandas
+    echo "DT_BUILD_NUMBER = $env:DT_BUILD_NUMBER"
 
-    pytest .\tests\
+    echo "DT_BUILD_SUFFIX = $env:DT_BUILD_SUFFIX"
 
 
 
-    rem Python 3.6
+    # Build and test wheel for Python 3.5

```

Comment:
Note that we're dropping python 3.5 support in the next release (#2642)

Reply:
Yeah, we can remove it at any time here.